cron/controller/main.go

```diff
@@ -140,7 +140,7 @@ func main() {
 	}
 	*metadata.NumShard = (shardNum + 1)
 	*metadata.ShardLoc = bucket + "/" + data.GetBlobFilename("", t)
-	*metadata.CommitSha = pkg.GetCommit()
+	*metadata.CommitSha = version.GetVersionInfo().GitCommit
```

Comment:
qq: what does the sigs.k8s.io/release-utils package provide that cannot be done using the simple code we had before?

Reply:
That'd be a question for #1696 I'd imagine. It mentions importing

But in my experience it seems to be a standardized way of setting certain version info

Reply:
I've asked in https://github.com/ossf/scorecard/pull/1881#issuecomment-1121277287